src/server/survey_and_program_data.R

```diff
@@ -162,13 +194,14 @@ surveyAndProgramData <- function(input, output, state, spectrumFilesState) {
 
     output$hot_program <- rhandsontable::renderRHandsontable({
         rhandsontable::rhandsontable(state$program_data_human_readable(), rowHeaders = NULL, stretchH = "all") %>%
-            rhandsontable::hot_col("Country or region", readOnly = TRUE) %>%
+            rhandsontable::hot_col("Country or region") %>%
```

Comment:
This is kind of scope creep of this issue but I wonder if we can make the same fix with the survey data. We're seeing the same problem where when editing in place if I add a new row the country is blank but I cannot edit the field.

Alternatively I'm happy to put in another ticket to fix this separately.

Reply:
I think lets open a new ticket for this, because at the moment we have no validation of survey data beyond validating that some exists. So lets open a new ticket for adding minimal validation of survey data (just checking country name) and making the editing in place actually work (i.e. setting relevant factors for age and sex, plus making country editable)!

https://vimc.myjetbrains.com/youtrack/issue/mrc-167